source/services/dialog/dialog.service.ts

```diff
@@ -58,6 +46,50 @@ export class DialogService<TDialogSettings> implements IDialogService<TDialogSet
 
 		return this.dialog.prompt(options, require('./promptDialog.html'));
 	}
+
+	openAutosaveForm(options: IAutosaveDialogSettings): void {
+		this.promise.resolvePromises(options.resolve).then((resolveData: any): void => {
+			let scope: IAutosaveDialogScope = <IAutosaveDialogScope>options.scope;
+
+			if (scope == null) {
+				scope = <IAutosaveDialogScope>this.$rootScope.$new();
```

Comment:
Nevermind, we don't have access to any scope in this context, so can't create a scope from the parent. By parent, I meant the calling context, but there is no way to do this.